websites/D/Deezer/presence.ts

```diff
@@ -114,12 +134,30 @@ presence.on("UpdateData", async () => {
       }
     } else {
       const [podcastLink] = document.querySelector<HTMLAnchorElement>(
-        "div.marquee-content"
-      ).children;
+          "div.marquee-content"
+        ).children as unknown as [HTMLAnchorElement, HTMLAnchorElement],
+        [, , , , , podcastID] = podcastLink.href.split("/");
       [presenceData.state, presenceData.details] = document
         .querySelector("div.marquee-content")
         .textContent.split(" · ");
-      presenceData.largeImageKey = "deezer";
+
+      podcastCoverID ??= podcastID;
+      podcastCoverURL ??= (
+        await (
+          await fetch(`https://api.deezer.com/podcast/${podcastCoverID}`)
+        ).json()
+      ).picture;
+
+      if (podcastCoverID !== podcastID) {
+        podcastCoverID = podcastID;
+        podcastCoverURL = (
+          await (
+            await fetch(`https://api.deezer.com/podcast/${podcastCoverID}`)
+          ).json()
+        ).picture;
+      }
+
+      presenceData.largeImageKey = cover ? podcastCoverURL : "deezer";
```

Comment:
`deezer` is already set on L9. Use the nullish operator instead
```suggestion
      presenceData.largeImageKey = cover ?? podcastCoverURL;
```